cpp/test/test_graph.cc

```diff
@@ -80,6 +80,31 @@ TEST_CASE("Graph") {
             it_begin.property<int64_t>("id").value());
   }
 
+  SECTION("ListProperty") {
+    // read file and construct graph info
+    std::string path =
+        root + "/ldbc_sample/parquet/ldbc_sample_with_feature.graph.yml";
+    auto maybe_graph_info = GraphInfo::Load(path);
+    REQUIRE(maybe_graph_info.status().ok());
+    auto graph_info = maybe_graph_info.value();
+    std::string label = "person", list_property = "feature";
+    auto maybe_vertices_collection =
+        VerticesCollection::Make(graph_info, label);
+    REQUIRE(!maybe_vertices_collection.has_error());
+    auto vertices = maybe_vertices_collection.value();
+    auto count = 0;
+    for (auto it = vertices->begin(); it != vertices->end(); ++it) {
+      auto vertex = *it;
+      auto float_array = vertex.property<FloatArray>(list_property).value();
```

Comment:
I think it could be better to acquire the datatype of list_property first before getting the value.

Reply:
done